CMakeLists.txt

```diff
@@ -50,9 +50,9 @@ OPTION (BUILD_UNIT_TESTS "Build the unitary tests" ON)
 
 FINDPYTHON()
 
-INCLUDE_DIRECTORIES(SYSTEM ${PYTHON_INCLUDE_DIRS})
+INCLUDE_DIRECTORIES(${PYTHON_INCLUDE_DIRS})
```

Comment:
Thinking about it more, I possibly should revert the change to Python for sure, but may leave for NumPy?

Reply:
You should indeed keep it here, and you can remove it for the pkgconfig part.